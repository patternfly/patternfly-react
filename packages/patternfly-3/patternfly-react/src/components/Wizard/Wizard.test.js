import React from 'react';
import { mount, shallow } from 'enzyme';
import { Wizard } from '../../index';

test('Wizard header renders properly', () => {
  const onCloseMock = jest.fn();
  const component = shallow(<Wizard.Header onClose={onCloseMock} title="Wizard Title" />);
  component.find('.close').simulate('click');
  expect(onCloseMock).toBeCalled();
  expect(component).toMatchSnapshot('Wizard header snapshot');
});

test('Wizard body renders properly', () => {
  const component = shallow(<Wizard.Body className="someAdditionalClass">Some Children</Wizard.Body>);
  expect(component).toMatchSnapshot('Wizard body snapshot');
});

test('Wizard steps renders properly', () => {
  const component = shallow(
    <Wizard.Steps
      className="someAdditionalClass"
      steps={[
        <Wizard.Step key={0} stepIndex={0} step={0} activeStep={0} label="Step 1" title="Step 1" onClick={jest.fn()}>
          <Wizard.SubStep subStep="1.1" title="Step 1.1" activeSubStep={0} />
          <Wizard.SubStep subStep="1.2" title="Step 1.2" activeSubStep={0} />
        </Wizard.Step>,
        <Wizard.Step key={1} stepIndex={1} step={1} label="Step 2" title="Step 2" activeStep={0} onClick={jest.fn()}>
          <Wizard.SubStep subStep="2.1" title="Step 2.1" activeSubStep={0} />
          <Wizard.SubStep subStep="2.2" title="Step 2.2" activeSubStep={0} />
        </Wizard.Step>
      ]}
    />
  );
  expect(component).toMatchSnapshot('Wizard steps snapshot');
});

test('Wizard step renders properly', () => {
  const component = shallow(
    <Wizard.Step key={0} stepIndex={0} step={0} activeStep={0} label="Step 1" title="Step 1" onClick={jest.fn()}>
      <Wizard.SubStep subStep="1.1" title="Step 1.1" activeSubStep={0} />
      <Wizard.SubStep subStep="1.2" title="Step 1.2" activeSubStep={0} />
    </Wizard.Step>
  );
  expect(component).toMatchSnapshot('Wizard step snapshot');
});

test('Wizard sub step renders properly', () => {
  const component = shallow(<Wizard.SubStep subStep="1.1" title="Step 1.1" activeSubStep={0} />);
  expect(component).toMatchSnapshot('Wizard sub step snapshot');
});

test('Wizard row renders properly', () => {
  const component = shallow(<Wizard.Row className="someAdditionalClass">Some Children</Wizard.Row>);
  expect(component).toMatchSnapshot('Wizard row snapshot');
});

test('Wizard sidebar renders properly', () => {
  const component = shallow(<Wizard.Sidebar className="someAdditionalClass" items={['item1', 'item2']} />);
  expect(component).toMatchSnapshot('Wizard sidebar snapshot');
});

test('Wizard sidebar group renders properly', () => {
  const component = shallow(
    <Wizard.SidebarGroup className="someAdditionalClass" step="1" activeStep="1">
      Some Children
    </Wizard.SidebarGroup>
  );
  expect(component).toMatchSnapshot('Wizard sidebar group snapshot');
});

test('Wizard sidebar group item renders properly', () => {
  const onClickMock = jest.fn();
  const stepIndex = 1;
  const subStepIndex = 1;

  const component = shallow(
    <Wizard.SidebarGroupItem
      className="wizard-sidebar-group-item"
      key="1.1"
      stepIndex={stepIndex}
      subStepIndex={subStepIndex}
      subStep="1.1"
      label="1A."
      title="Details"
      activeSubStep="1.1"
      onClick={onClickMock}
    />
  );
  component.find('a').simulate('click', { preventDefault() {} });
  expect(onClickMock).toBeCalledWith(stepIndex, subStepIndex);
  expect(component).toMatchSnapshot('Wizard sidebar group item snapshot');
});

test('Wizard main renders properly', () => {
  const component = shallow(<Wizard.Main className="someAdditionalClass">Some Children</Wizard.Main>);
  expect(component).toMatchSnapshot('Wizard main snapshot');
});

test('Wizard contents renders properly', () => {
  const component = shallow(
    <Wizard.Contents
      className="someAdditionalClass"
      stepIndex={1}
      subStepIndex={1.1}
      activeStepIndex={1}
      activeSubStepIndex={1.1}
    >
      Some Children
    </Wizard.Contents>
  );
  expect(component).toMatchSnapshot('Wizard contents snapshot');
});

test('Wizard contents renders hidden class when not active', () => {
  const component = shallow(
    <Wizard.Contents
      className="someAdditionalClass"
      stepIndex={1}
      subStepIndex={1.1}
      activeStepIndex={1}
      activeSubStepIndex={1.2}
    >
      Some Children
    </Wizard.Contents>
  );
  expect(component).toMatchSnapshot('Wizard contents hidden snapshot');
});

test('Wizard review steps renders properly', () => {
  const component = shallow(<Wizard.ReviewSteps className="someAdditionalClass">Some Children</Wizard.ReviewSteps>);
  expect(component).toMatchSnapshot('Wizard review steps snapshot');
});

test('Wizard review step renders properly', () => {
  const onClickMock = jest.fn();
  const component = shallow(
    <Wizard.ReviewStep onClick={onClickMock} title="Step One" collapsed>
      Some Children
    </Wizard.ReviewStep>
  );
  component.find('a').simulate('click');
  expect(onClickMock).toBeCalled();
  expect(component).toMatchSnapshot('Wizard review step snapshot');
});

test('Wizard review sub steps renders properly', () => {
  const component = shallow(
    <Wizard.ReviewSubSteps className="someAdditionalClass" collapsed>
      Some Children
    </Wizard.ReviewSubSteps>
  );
  expect(component).toMatchSnapshot('Wizard review sub steps snapshot');
});

test('Wizard review sub step renders properly', () => {
  const onClickMock = jest.fn();
  const component = shallow(
    <Wizard.ReviewSubStep onClick={onClickMock} label="1.1" title="1.1" collapsed>
      Some Children
    </Wizard.ReviewSubStep>
  );
  component.find('a').simulate('click');
  expect(onClickMock).toBeCalled();
  expect(component).toMatchSnapshot('Wizard review sub step snapshot');
});

test('Wizard review content renders properly', () => {
  const component = shallow(
    <Wizard.ReviewContent className="someAdditionalClass" collapsed>
      Some Children
    </Wizard.ReviewContent>
  );
  expect(component).toMatchSnapshot('Wizard review content snapshot');
});

test('Wizard review item renders properly', () => {
  const component = shallow(<Wizard.ReviewItem className="someAdditionalClass">Some Children</Wizard.ReviewItem>);
  expect(component).toMatchSnapshot('Wizard review item snapshot');
});

test('Wizard footer renders properly', () => {
  const component = shallow(<Wizard.Footer className="someAdditionalClass">Some Children</Wizard.Footer>);
  expect(component).toMatchSnapshot('Wizard footer snapshot');
});

test('Wizard pattern body renders properly while loading', () => {
  const component = shallow(
    <Wizard.Pattern.Body loading loadingTitle="Wizard Title" loadingMessage="Wizard loading..." />
  );
  expect(component).toMatchSnapshot('Wizard pattern body loading snapshot');
});

test('Wizard pattern body renders active step', () => {
  const component = shallow(
    <Wizard.Pattern.Body
      loading={false}
      steps={[
        { title: 'General', render: () => <p>General</p> },
        { title: 'Step Two', render: () => <p>Step Two</p> },
        { title: 'Step Three', render: () => <p>Step Three</p> }
      ]}
      activeStepIndex={0}
    />
  );
  expect(component).toMatchSnapshot('Wizard pattern body renders active step snapshot');
});

test('Wizard pattern body renders empty if no steps and not loading', () => {
  const component = shallow(<Wizard.Pattern.Body loading={false} />);
  expect(component).toMatchSnapshot('Wizard pattern body renders empty snapshot');
});

const testWizardPattern = props => {
  const onHide = jest.fn();
  const onExited = jest.fn();
  const onStepChanged = jest.fn();
  return (
    <Wizard.Pattern
      show
      onHide={onHide}
      onExited={onExited}
      title="Wizard Pattern Example"
      nextStepDisabled={false}
      steps={[
        { title: 'General', render: () => <p>General</p> },
        { title: 'Step Two', render: () => <p>Step Two</p> },
        { title: 'Step Three', render: () => <p>Step Three</p> }
      ]}
      loadingTitle="Loading..."
      loadingMessage="This may take a minute."
      activeStepIndex={0}
      onStepChanged={onStepChanged}
      {...props}
    />
  );
};

test('Wizard Pattern renders properly', () => {
  const component = shallow(testWizardPattern({ loading: false }));
  expect(component).toMatchSnapshot('Wizard pattern snapshot');
});

test('Wizard Pattern hides onClose', () => {
  const onHideMock = jest.fn();

  const component = mount(
    testWizardPattern({
      onHide: onHideMock
    })
  );
  component.find('.close').simulate('click');

  expect(onHideMock).toBeCalled();
});

test('Wizard Pattern sets active step to step 2 after step click', () => {
  const onStepChangedMock = jest.fn();
  const onNextMock = jest.fn();

  const component = mount(
    testWizardPattern({
      onStepChanged: onStepChangedMock,
      onNext: onNextMock
    })
  );
  component
    .find('.wizard-pf-step a')
    .at(1)
    .simulate('click');

  expect(onNextMock).toBeCalledWith(1);
  expect(onStepChangedMock).toBeCalledWith(1);
});

test('Wizard Pattern accepts next step then previous step', () => {
  let eventCount = 0;
  const onBackMock = jest.fn();

  const component = mount(
    testWizardPattern({
      onStepChanged: () => eventCount++,
      onBack: onBackMock
    })
  );

  // click next
  component
    .find('.wizard-pf-footer button')
    .at(2)
    .simulate('click');

  component.setProps({ activeStepIndex: 1 });

  // click previous
  component
    .find('.wizard-pf-footer button')
    .at(1)
    .simulate('click');

  expect(onBackMock).toBeCalledWith(0);
  expect(eventCount).toBe(2);
});

const testStatefulWizardPattern = props => {
  const onHide = jest.fn();
  const onExited = jest.fn();
  const onStepChanged = jest.fn();
  return (
    <Wizard.Pattern.Stateful
      show
      onHide={onHide}
      onExited={onExited}
      title="Wizard Pattern Stateful Example"
      shouldDisableNextStep={() => false}
      shouldDisablePreviousStep={() => false}
      shouldPreventStepChange={() => false}
      steps={[
        { title: 'General', render: () => <p>General</p> },
        { title: 'Step Two', render: () => <p>Step Two</p> },
        { title: 'Step Three', render: () => <p>Step Three</p> }
      ]}
      loadingTitle="Loading..."
      loadingMessage="This may take a minute."
      onStepChanged={onStepChanged}
      {...props}
    />
  );
};

test('Wizard Stateful Pattern renders properly', () => {
  const component = shallow(testStatefulWizardPattern());
  expect(component.state().activeStepIndex).toBe(0);
  expect(component).toMatchSnapshot('Wizard stateful pattern snapshot');
});

test('Wizard Stateful Pattern renders properly after wizard step click', () => {
  const component = mount(testStatefulWizardPattern());
  component
    .find('.wizard-pf-step a')
    .at(1)
    .simulate('click');

  expect(component.state().activeStepIndex).toBe(1);
});

test('Wizard Stateful Pattern should return on shouldPreventStepChange', () => {
  const component = mount(
    testStatefulWizardPattern({
      shouldPreventStepChange: (activeStepIndex, newStepIndex) => true
    })
  );
  component
    .find('.wizard-pf-step a')
    .at(1)
    .simulate('click');

  expect(component.state().activeStepIndex).toBe(0);
});

const testDisableNextStepWizard = props => {
  const onHide = jest.fn();
  const onExited = jest.fn();
  const onStepChanged = jest.fn();
  return (
    <Wizard.Pattern.Stateful
      show
      onHide={onHide}
      onExited={onExited}
      onStepChanged={onStepChanged}
      title="Wizard Disable Next Step"
      shouldDisableNextStep={() => true}
      steps={[
        { title: '1', render: () => <p>1</p> },
        { title: '2', render: () => <p>2</p> },
        { title: '3', render: () => <p>3</p> }
      ]}
      {...props}
    />
  );
};

const testDisablePreviousStepWizard = props => {
  const onHide = jest.fn();
  const onExited = jest.fn();
  const onStepChanged = jest.fn();
  return (
    <Wizard.Pattern.Stateful
      show
      onHide={onHide}
      onExited={onExited}
      onStepChanged={onStepChanged}
      title="Wizard Disable Next Step"
      shouldDisablePreviousStep={() => true}
      steps={[
        { title: '1', render: () => <p>1</p> },
        { title: '2', render: () => <p className=".step2">2</p> },
        { title: '3', render: () => <p>3</p> }
      ]}
      {...props}
    />
  );
};

test('Wizard Stateful with shouldDisableNextStep should disable next step', () => {
  const component = mount(testDisableNextStepWizard());
  expect(
    component
      .find('.wizard-pf-footer .btn')
      .at(2)
      .getDOMNode().disabled
  ).toBe(true);
});

test('Wizard Stateful with shouldDisablePreviousStep should disable previous step', () => {
  const component = mount(testDisablePreviousStepWizard());
  component
    .find('.wizard-pf-footer .btn')
    .at(2)
    .simulate('click');
  expect(component.exists('.step2')).toEqual(true);
  expect(
    component
      .find('.wizard-pf-footer .btn')
      .at(1)
      .getDOMNode().disabled
  ).toBe(true);
});
