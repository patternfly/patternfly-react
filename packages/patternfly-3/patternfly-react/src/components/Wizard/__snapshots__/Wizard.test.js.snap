// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wizard Pattern renders properly: Wizard pattern snapshot 1`] = `
<Wizard
  className=""
  dialogClassName=""
  onClose={[Function]}
  onExited={[MockFunction]}
  onHide={[Function]}
  show={true}
>
  <WizardHeader
    onClose={[Function]}
    title="Wizard Pattern Example"
  />
  <WizardBody
    className=""
  >
    <WizardPatternBody
      activeStepIndex={0}
      activeStepStr="1"
      goToStep={[Function]}
      loading={false}
      loadingMessage="This may take a minute."
      loadingTitle="Loading..."
      nextStepDisabled={false}
      stepButtonsDisabled={false}
      steps={
        Array [
          Object {
            "render": [Function],
            "title": "General",
          },
          Object {
            "render": [Function],
            "title": "Step Two",
          },
          Object {
            "render": [Function],
            "title": "Step Three",
          },
        ]
      }
    />
  </WizardBody>
  <WizardFooter
    className=""
  >
    <Button
      active={false}
      block={false}
      bsClass="btn"
      bsStyle="default"
      className="btn-cancel"
      disabled={false}
      onClick={[Function]}
    >
      Cancel
    </Button>
    <Button
      active={false}
      block={false}
      bsClass="btn"
      bsStyle="default"
      disabled={true}
      onClick={[Function]}
    >
      <Icon
        name="angle-left"
        type="fa"
      />
      Back
    </Button>
    <Button
      active={false}
      block={false}
      bsClass="btn"
      bsStyle="primary"
      disabled={false}
      onClick={[Function]}
    >
      <React.Fragment>
        Next
        <Icon
          name="angle-right"
          type="fa"
        />
      </React.Fragment>
    </Button>
  </WizardFooter>
</Wizard>
`;

exports[`Wizard Stateful Pattern renders properly: Wizard stateful pattern snapshot 1`] = `
<WizardPattern
  activeStepIndex={0}
  backText="Back"
  bodyHeader={null}
  cancelText="Cancel"
  closeText="Close"
  loading={false}
  loadingMessage="This may take a minute."
  loadingTitle="Loading..."
  nextButtonRef={[Function]}
  nextStepDisabled={false}
  nextText="Next"
  onBack={[Function]}
  onExited={[MockFunction]}
  onHide={[MockFunction]}
  onNext={[Function]}
  onStepChanged={[Function]}
  previousStepDisabled={false}
  show={true}
  stepButtonsDisabled={false}
  steps={
    Array [
      Object {
        "render": [Function],
        "title": "General",
      },
      Object {
        "render": [Function],
        "title": "Step Two",
      },
      Object {
        "render": [Function],
        "title": "Step Three",
      },
    ]
  }
  title="Wizard Pattern Stateful Example"
/>
`;

exports[`Wizard body renders properly: Wizard body snapshot 1`] = `
<ModalBody
  bsClass="modal-body"
  className="wizard-pf-body clearfix someAdditionalClass"
  componentClass="div"
>
  Some Children
</ModalBody>
`;

exports[`Wizard contents renders hidden class when not active: Wizard contents hidden snapshot 1`] = `
<div
  className="wizard-pf-contents hidden someAdditionalClass"
>
  Some Children
</div>
`;

exports[`Wizard contents renders properly: Wizard contents snapshot 1`] = `
<div
  className="wizard-pf-contents someAdditionalClass"
>
  Some Children
</div>
`;

exports[`Wizard footer renders properly: Wizard footer snapshot 1`] = `
<ModalFooter
  bsClass="modal-footer"
  className="wizard-pf-footer someAdditionalClass"
  componentClass="div"
>
  Some Children
</ModalFooter>
`;

exports[`Wizard header renders properly: Wizard header snapshot 1`] = `
<ModalHeader
  bsClass="modal-header"
  closeButton={false}
  closeLabel="Close"
>
  <button
    aria-hidden="true"
    aria-label="Close"
    className="close"
    onClick={
      [MockFunction] {
        "calls": Array [
          Array [],
        ],
      }
    }
  >
    <Icon
      name="close"
      type="pf"
    />
  </button>
  <ModalTitle
    bsClass="modal-title"
    componentClass="h4"
  >
    Wizard Title
  </ModalTitle>
</ModalHeader>
`;

exports[`Wizard main renders properly: Wizard main snapshot 1`] = `
<div
  className="wizard-pf-main someAdditionalClass"
>
  Some Children
</div>
`;

exports[`Wizard pattern body renders active step: Wizard pattern body renders active step snapshot 1`] = `
<React.Fragment>
  <WizardSteps
    className=""
    steps={
      Array [
        <WizardStep
          activeStep="1"
          className=""
          label="1"
          onClick={[Function]}
          step="1"
          stepIndex={0}
          title="General"
        />,
        <WizardStep
          activeStep="1"
          className=""
          label="2"
          onClick={[Function]}
          step="2"
          stepIndex={1}
          title="Step Two"
        />,
        <WizardStep
          activeStep="1"
          className=""
          label="3"
          onClick={[Function]}
          step="3"
          stepIndex={2}
          title="Step Three"
        />,
      ]
    }
  />
  <WizardRow
    className=""
  >
    <WizardMain
      className=""
    >
      <WizardContents
        activeStepIndex={0}
        activeSubStepIndex={null}
        className=""
        stepIndex={0}
        subStepIndex={null}
      >
        <p>
          General
        </p>
      </WizardContents>
    </WizardMain>
  </WizardRow>
</React.Fragment>
`;

exports[`Wizard pattern body renders empty if no steps and not loading: Wizard pattern body renders empty snapshot 1`] = `""`;

exports[`Wizard pattern body renders properly while loading: Wizard pattern body loading snapshot 1`] = `
<WizardRow
  className=""
>
  <WizardMain
    className=""
  >
    <EmptyState
      className=""
    >
      <Spinner
        className="blank-slate-pf-icon"
        inline={false}
        inverse={false}
        loading={true}
        size="lg"
      />
      <EmptyStateAction
        className=""
        secondary={false}
      >
        <h3>
          Wizard Title
        </h3>
      </EmptyStateAction>
      <EmptyStateAction
        className=""
        secondary={true}
      >
        <p>
          Wizard loading...
        </p>
      </EmptyStateAction>
    </EmptyState>
  </WizardMain>
</WizardRow>
`;

exports[`Wizard review content renders properly: Wizard review content snapshot 1`] = `
<div
  className="wizard-pf-review-content collapse someAdditionalClass"
>
  Some Children
</div>
`;

exports[`Wizard review item renders properly: Wizard review item snapshot 1`] = `
<div
  className="wizard-pf-review-item someAdditionalClass"
>
  Some Children
</div>
`;

exports[`Wizard review step renders properly: Wizard review step snapshot 1`] = `
<ListGroupItem
  bsClass="list-group-item"
  listItem={true}
>
  <a
    className="collapsed"
    href="#"
    onClick={
      [MockFunction] {
        "calls": Array [
          Array [],
        ],
      }
    }
  >
    Step One
  </a>
  Some Children
</ListGroupItem>
`;

exports[`Wizard review steps renders properly: Wizard review steps snapshot 1`] = `
<div
  className="wizard-pf-review-steps someAdditionalClass"
>
  <ListGroup
    bsClass="list-group"
    componentClass="ul"
  >
    Some Children
  </ListGroup>
</div>
`;

exports[`Wizard review sub step renders properly: Wizard review sub step snapshot 1`] = `
<ListGroupItem
  bsClass="list-group-item"
  listItem={true}
>
  <a
    className="collapsed"
    href="#"
    onClick={
      [MockFunction] {
        "calls": Array [
          Array [],
        ],
      }
    }
  >
    <span
      className="wizard-pf-substep-number"
    >
      1.1
    </span>
    <span
      className="wizard-pf-substep-title"
    >
      1.1
    </span>
  </a>
  Some Children
</ListGroupItem>
`;

exports[`Wizard review sub steps renders properly: Wizard review sub steps snapshot 1`] = `
<div
  className="wizard-pf-review-substeps collapse someAdditionalClass"
>
  <ListGroup
    bsClass="list-group"
    componentClass="ul"
  >
    Some Children
  </ListGroup>
</div>
`;

exports[`Wizard row renders properly: Wizard row snapshot 1`] = `
<section
  className="wizard-pf-row someAdditionalClass"
>
  Some Children
</section>
`;

exports[`Wizard sidebar group item renders properly: Wizard sidebar group item snapshot 1`] = `
<ListGroupItem
  bsClass="list-group-item"
  className="active wizard-sidebar-group-item"
  listItem={true}
>
  <a
    href="#"
    onClick={[Function]}
  >
    <span
      className="wizard-pf-substep-number"
    >
      1A.
    </span>
    <span
      className="wizard-pf-substep-title"
    >
      Details
    </span>
  </a>
</ListGroupItem>
`;

exports[`Wizard sidebar group renders properly: Wizard sidebar group snapshot 1`] = `
<ListGroup
  bsClass="list-group"
  className="someAdditionalClass"
  componentClass="ul"
>
  Some Children
</ListGroup>
`;

exports[`Wizard sidebar renders properly: Wizard sidebar snapshot 1`] = `
<div
  className="wizard-pf-sidebar someAdditionalClass"
>
  item1
  item2
</div>
`;

exports[`Wizard step renders properly: Wizard step snapshot 1`] = `
<li
  className="wizard-pf-step active"
>
  <a
    href="#"
    onClick={[Function]}
  >
    <span
      className="wizard-pf-step-number"
    >
      Step 1
    </span>
    <span
      className="wizard-pf-step-title"
    >
      Step 1
    </span>
    <WizardSubStep
      activeSubStep={0}
      className=""
      subStep="1.1"
      title="Step 1.1"
    />
    <WizardSubStep
      activeSubStep={0}
      className=""
      subStep="1.2"
      title="Step 1.2"
    />
  </a>
</li>
`;

exports[`Wizard steps renders properly: Wizard steps snapshot 1`] = `
<div
  className="wizard-pf-steps someAdditionalClass"
>
  <ul
    className="wizard-pf-steps-indicator"
  >
    <WizardStep
      activeStep={0}
      className=""
      key="0"
      label="Step 1"
      onClick={[MockFunction]}
      step={0}
      stepIndex={0}
      title="Step 1"
    >
      <WizardSubStep
        activeSubStep={0}
        className=""
        subStep="1.1"
        title="Step 1.1"
      />
      <WizardSubStep
        activeSubStep={0}
        className=""
        subStep="1.2"
        title="Step 1.2"
      />
    </WizardStep>
    <WizardStep
      activeStep={0}
      className=""
      key="1"
      label="Step 2"
      onClick={[MockFunction]}
      step={1}
      stepIndex={1}
      title="Step 2"
    >
      <WizardSubStep
        activeSubStep={0}
        className=""
        subStep="2.1"
        title="Step 2.1"
      />
      <WizardSubStep
        activeSubStep={0}
        className=""
        subStep="2.2"
        title="Step 2.2"
      />
    </WizardStep>
  </ul>
</div>
`;

exports[`Wizard sub step renders properly: Wizard sub step snapshot 1`] = `
<span
  className="wizard-pf-step-title-substep"
>
  Step 1.1
</span>
`;
